plugins/tailwind/compiler.ts

```diff
@@ -76,3 +97,44 @@ export async function initTailwind(
 
   return postcss(plugins);
 }
+
+const fileTypes = [".tsx"];
+
+function extractSpecifiers(graph: ModuleGraphJson, projectLocation: string) {
+  const specifiers: Set<string> = new Set();
+
+  for (const module of graph.modules) {
+    if (
+      fileTypes.some((type) =>
+        module.specifier.endsWith(type) &&
+        module.specifier.startsWith(path.dirname(projectLocation))
+      )
+    ) {
+      specifiers.add(module.specifier);
+    }
+  }
+  return Array.from(specifiers);
+}
```

Comment:
```suggestion
function extractSpecifiers(graph: ModuleGraphJson, projectLocation: string) {
  return graph.modules
    .filter((module) =>
      module.specifier.endsWith(".tsx") &&
      module.specifier.startsWith(path.dirname(projectLocation))
    )
    .map((module) => module.specifier);
}
```
Nit: `fileTypes` as an array doesn't appear needed. I might be missing context, though.

Reply:
Great simplification. Thanks.